### PR TITLE
Create transaction table

### PR DESCRIPTION
**Pull request by BenjaminParment**
-> hooks transactions to redux with dummy data
-> Creates table to display transactions
-> Creates transaction page
-> Creates statusCell just like the one for ErrorCell of submissions
     -> That is using a switch to determine what status should be displayed, this will be changed once we know the structure of objects coming back from the backend

### CODE REVIEW

File: src/shared/utils/formatters.test.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { Column, RowNode } from "ag-grid-community";
+
+import { FormatterParams } from "components/reusables/table/Table";
+import { formatDate } from "./formatters";
+import moment from "moment";
+
+export const createParams = <T>(defaultTestParams: FormatterParams<T>, value: unknown, data?: Partial<T>) => ({
```
Comment (BenjaminParment) [inline src/shared/utils/formatters.test.ts]: idk if this was the best thing to do, should I move this outside of the test file?

File: src/components/transactions/transactionsFormatters.ts [hunk 2]
```diff
@@ -1,2 +1,2 @@
 export const formatTransactionsNumber = (dataSize?: number): string =>
-    `${dataSize} transaction${dataSize === 1 ? "" : "s"}`;
+    dataSize ? `${dataSize} transaction${dataSize === 1 ? "" : "s"}` : "0 transactions";
```
Comment (j-n-white) [inline src/components/transactions/transactionsFormatters.ts]:
```suggestion
    `${dataSize || 0} transaction${dataSize === 1 ? "" : "s"}`;

```
This is slightly neater

File: src/shared/utils/formatters.test.ts (hunk 1 above)
Comment (j-n-white) [reply src/shared/utils/formatters.test.ts]: This is fine. However you don't need to pass the default in as an argument just declare the const beforehand and reference that.

**Comment by BenjaminParment** [comment]
For the createParams comment:
Am I not supposed to pass down the defaultParams to test the submissions and transactions formatter? since they both need their own different test params

**Pull request merged**